include/alpaka/event/EventUniformCudaHipRt.hpp

```diff
@@ -101,6 +106,10 @@ namespace alpaka
         {
             return !((*this) == rhs);
         }
+        auto getNativeHandle() const noexcept
```

Comment:
```suggestion

        [[nodiscard]] auto getNativeHandle() const noexcept
```

Note the extra linebreak.